recipes/pretty_html_table/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pretty_html_table" %}
+{% set version = "0.9.dev0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "63b3ce16ab1abdbc4888094ba792908666122eafe3fd98ed127efbb4dd3ac0f9"
+
+build:
```

Comment:
Sorry I missed this in the initial review. I think this can be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python